R/000-wrappers.R

```diff
@@ -37,16 +37,16 @@ NULL
   stop(class, " cannot be modified", call. = FALSE)
 }
 
-#' Calculate the sum of a vector of integers using multiple threads.
+#' Calculate the sum of a vector of real numbers using multiple threads.
 #'
-#' @param x A vector of integers to sum over.
+#' @param x A vector of real numbers to sum over.
 #' @param n The number of threads used to compute this calculation (int).
 #'
 #' @return The sum of all elements of the input vector.
 #'
 #' @export
```

Comment:
We should consider not exporting this. I see this as an internal function that users of the package don't need to know about.

Reply:
Ah oops! Totally agreed